invisible_cities/reco/calib_functions.py

```diff
@@ -125,21 +125,19 @@ def valid_integral_limits(sample_width, n_integrals, integral_start, integral_wi
             filter_limits(anti, buffer_length))
 
 
-def copy_sensor_table(h5in, h5out):
-    # Copy sensor table if exists (needed for Non DB calibrations)
+def copy_sensor_table(h5in_name : str,
+                      h5out     : tb.file.File):
 
-    with tb.open_file(h5in) as dIn:
-        if 'Sensors' not in dIn.root:
-            return
-        group    = h5out.create_group(h5out.root, "Sensors")
-
-        if 'DataPMT' in dIn.root.Sensors:
-            datapmt  = dIn.root.Sensors.DataPMT
-            datapmt.copy(newparent=group)
-
-        if 'DataSiPM' in dIn.root.Sensors:
-            datasipm = dIn.root.Sensors.DataSiPM
-            datasipm.copy(newparent=group)
+    with tb.open_file(h5in_name) as dIn:
+        try:
+            sensor_info = dIn.root.Sensors
+        except tb.exceptions.NoSuchNodeError:
+            sensor_info = None
+
+        if sensor_info:
+            h5out.copy_node(sensor_info,
+                            newparent = h5out.root,
+                            recursive = True)
 
```

Comment:
You can "merge" the `try` and `if` statements as:

```
with tb.open_file(h5in_name) as dIn:
    try:
        sensor_info = dIn.root.Sensors
        h5out.copy_node(sensor_info,
                        newparent = h5out.root,
                        recursive = True)
    except tb.exceptions.NoSuchNodeError:
        pass
```

I would like a info message in the except part, but I don't know how to test it... I think we have had this problem already.